Cargo.toml

```diff
@@ -49,7 +49,7 @@ env_logger = "0.8"
 mockito = "0.29"
 spectral = "0.6"
 test-case = "1.0.0"
-tokio = { version = "0.2", features = ["macros"] }
+tokio = { version = "1.0", features = ["full"] }
```

Comment:
Is there maybe an equivalent of `"macros"`? I suspect we don't need the full featureset in this crate.

Reply:
AFAIK macros are still gated on the `macros` feature.